daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -28,6 +28,7 @@
 // * 5 -- 2019-05-22: Relax serializability constraints for contract ids
 //        2019-05-23: Add BuiltinFunction.COERCE_CONTRACT_ID
 //        2019-05-24: Make actors in exercise optional
+//        2019-05-27: Add enum type.
```

Comment:
Is this true? Shouldn't this go in `dev`?

Reply:
fixed